chat-plugins/info.js

```diff
@@ -264,46 +264,52 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim()) return this.parse(`/help ipsearch`);
 		if (!this.can('rangeban')) return;
-		let results = [];
 
+		target = target.split(',');
```

Comment:
...no, `target` should be a string, not an array.

There's already `this.splitTargetText`...